.github/workflows/base-image.yaml

```diff
@@ -10,6 +10,12 @@ jobs:
     name: build mini-lab-sonic base image
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        flavors:
```

Comment:
"flavor" is already used in the mini-lab to differentiate between different deployment setups. Maybe we can just use something like "version" here in order not to mix it up?